packages/nimble-components/src/tooltip/tests/tooltip.spec.ts

```diff
@@ -20,7 +28,13 @@ describe('Tooltip', () => {
         if (!iconElement) {
             return false;
         }
-        return window.getComputedStyle(iconElement).display !== 'none';
+        const display = window.getComputedStyle(iconElement).display;
+        if (display === '') {
+            throw new Error('Value of display was unexpectedly empty');
+        }
+        return (
+            display === 'block' || display === 'inline' || display === 'flex'
+        );
```

Comment:
Do we need the parentheses here?
```suggestion
        return display === 'block' || display === 'inline' || display === 'flex';
```

Reply:
Fixed

Reply:
Ugh. Prettier adds them back the way they were. 😡